python_modules/dagster-airflow/dagster_airflow/factory.py

```diff
@@ -60,23 +60,77 @@ def python_callable(**kwargs):
             step_keys=json.dumps(step_keys),
             pipeline_name=pipeline.name,
         )
-        return execute_query_from_cli(repository_container, query, variables=None)
+        return json.loads(execute_query_from_cli(repository_container, query, variables=None))
 
     return python_callable
 
 
-def make_airflow_dag(pipeline, env_config=None, dag_id=None, dag_description=None, dag_kwargs=None):
+def _make_python_operator(pipeline, env_config, solid_name, step_keys, dag, dag_id, op_kwargs):
```

Comment:
We might want to consider more officially documenting this method via an abstractclassmethod of some sort. One slight downside is the we would need a DagsterPythonOperator subclass and then a mixin marking something as a DagsterOperator that has a @abstractclass method with 
(pipeline, env_config, solid_name, step_keys, dag, dag_id, op_kwargs) args

Then _make_airflow_dag could take a class as an argument with an is_subclass check and it would call that class method directly

Reply:
This will make it clear how to plugin a future K8s or operator or whatever in when the time comes

Reply:
I'm totally fine with this approach

Reply:
Implemented this in c48492b